assignments/arrays.js

```diff
@@ -80,17 +87,29 @@ console.log();
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
+  for(let i = 0; i < inventory.length; i++) {
+    years = inventory.car_year;
+    carYears.push(years);
+  }
 console.log();
 
 // ==== Challenge 5 ====
 // The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
 let oldCars =[];
-console.log(); 
+  for (let i = 0; i < carYears.length; i++) {
+    if (car_year < 2000) {
+      old = car_year
+      oldCars.push(old);
+    }
+  }
+console.log();
 
 // ==== Challenge 6 ====
 // A buyer is interested in seeing only BMW and Audi cars within the inventory.  Return an array that only contains BMW and Audi cars.  Once you have populated the BMWAndAudi array, use JSON.stringify() to show the results of the array in the console.
 let BMWAndAudi =[];
-console.log();
-
-
+  for(i = 0; inventory.length; i++)
+    if(car_make === BMW || Audi) {
```

Comment:
`car_make` doesn't refer to anything at this point. We need to get it from the element in the array.
```
inventory[i].car_make === 'BMW'
```
Remember to make `BMW` a string.

And another annoying thing about `||` and `&&` operators is that you have to type out the whole thing again. So you would need the condition to be
```
inventory[i].car_make === 'BMW' || inventory[i].car_make === 'Audi'
```